<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreDialog</class>
 <widget class="QDialog" name="ScoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Score Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="titleEdit">
     <property name="toolTip">
      <string>The title of the score</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>The title of the score</string>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="toolTip">
      <string>The score's artist</string>
     </property>
     <property name="text">
      <string>Artist</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>The name of the person who wrote this score</string>
     </property>
     <property name="text">
      <string>Tabbed by</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="artistVisible">
     <property name="toolTip">
      <string>Show/hide artist name</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="artistEdit">
     <property name="toolTip">
      <string>The score's artist</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="creatorEdit">
     <property name="toolTip">
      <string>The name of the person who wrote this score</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="creatorVisible">
     <property name="toolTip">
      <string>Show.hide tabber name</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Swing?</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="swingComboBox">
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8ths</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16ths</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32nds</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="bpmSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Beats per minute of this score</string>
       </property>
       <property name="statusTip">
        <string>Beats per minute</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> bpm</string>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Beats per minute of this score</string>
     </property>
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="bpmVisible">
     <property name="toolTip">
      <string>Show/hide BPM</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>artistVisible</tabstop>
  <tabstop>artistEdit</tabstop>
  <tabstop>creatorVisible</tabstop>
  <tabstop>creatorEdit</tabstop>
  <tabstop>bpmVisible</tabstop>
  <tabstop>bpmSpinBox</tabstop>
  <tabstop>swingComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScoreDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScoreDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>artistVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>artistEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>creatorVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>creatorEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bpmVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>bpmSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
